### PR TITLE
Ports moths, adds some hippie linkers to tg code, mainly for preferen…

### PR DESCRIPTION
**Pull request by Carbonhell**
…ces and specie bodyparts

:cl: Ports moths!
add: Ports moths, along with their main feature(wings!)
/:cl:
[why]: It's been voted so

This also adds some technical proc hookups to tg code which will probs make fuckin with species easier in future
i sadly had to directly edit some tg files to make everything work properly, i tried to keep the edits at the minimum possible

### CODE REVIEW

**Comment by Carbonhell** [comment]
Stuff i'm unsure about:
1) Random body button resets wings to None atm, mainly because idk what happens if i set them to something,not sure if it impacts other races too
2) I am not 100% sure the preferences save works completely, i'd like someone else's opinion, and i will also try to run some local tests for the saves to be sure they're fully good

File: hippiestation/code/modules/mob/living/carbon/human/species_types/moth.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		randname += " [lastname]"
+
+	return randname
+
+/datum/species/moth/qualifies_for_rank(rank, list/features)
+	if(rank in GLOB.command_positions)
+		return 0
+	return 1
+
+// Procs edited on tg files:
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/carbon/human/species_types/moth.dm]: Document this in a `.md` file inside the `hippiestation` folder maybe a `changes.dm` file?

**Comment by Carbonhell** [comment]
it's ready

**Review by j-mie — APPROVED** [review]

File: hippiestation/code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -24,13 +24,80 @@
 	color_src = null
 	locked = TRUE
 
+//moth shit
+/datum/sprite_accessory/moth_wings
```
Comment (tortellinitony) [inline hippiestation/code/modules/mob/dead/new_player/sprite_accessories.dm:28]: I'd suggest you make this one "Plain"

**Review by tortellinitony — DISMISSED** [review]

File: hippiestation/changes.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Differences with /tg/
+
+The best practice to have while coding is to keep everything modularized, or the most possible. Sometimes, through,
+ it's not possible and you're required to put an "hookup" proc call in a tg file. This file is intended to have a list
+of those hookups, to not forget their locations and such.
+
+##Hookups list
```
Comment (tortellinitony) [inline hippiestation/changes.md]: Must be a space between ## and text

**Review by tortellinitony — CHANGES_REQUESTED** [review]

**Review by tortellinitony — APPROVED** [review]

**Comment by GuyonBroadway** [comment]
MOWTHES!!!

**Pull request merged**